lib/procore-sift.rb

```diff
@@ -39,20 +39,28 @@ def filter_errors
   def filter_validator
     @_filter_validator ||= FilterValidator.build(
       filters: filters,
-      sort_fields: self.class.sort_fields,
+      sort_fields: sort_fields,
       filter_params: filter_params,
       sort_params: sort_params,
     )
   end
 
   def filters
-    self.class.filters
+    self.class.ancestors.flat_map do |klass|
+      klass.try(:filters)
+    end.compact.uniq do |f|
+      [f.param, f.class]
+    end
```

Comment:
NB: just for the sake of syntax
```suggestion
    filters = self.class.ancestors.flat_map { |klass| klass.try(:filters) }
    filters.compact.uniq { |f| [f.param, f.class] }
```
or 
```suggestion
    self.class.ancestors.
      flat_map { |klass| klass.try(:filters) }.
      compact.
      uniq { |f| [f.param, f.class] }
```